spec/system/support_interface/reasons_for_rejection_dashboard_spec.rb

```diff
@@ -0,0 +1,161 @@
+    allow(EndOfCycleTimetable).to receive(:apply_reopens).and_return(60.days.ago)
+    application_choice1 = create(:application_choice, :awaiting_provider_decision)
+    application_choice2 = create(:application_choice, :awaiting_provider_decision)
+    application_choice3 = create(:application_choice, :awaiting_provider_decision)
+
+    Timecop.freeze(@today - 40.days) do
+      reject_application_with_structured_reasons(application_choice1)
+    end
+
+    Timecop.freeze(@today) do
+      reject_application_with_structured_reasons(application_choice2)
+      reject_application_without_structured_reasons(application_choice3)
+    end
+  end
+
+  def when_i_visit_the_performance_page_in_support
+    visit support_interface_performance_path
+  end
+
+  def and_i_click_on_the_reasons_for_rejection_metrics_link
+    click_on 'Reasons for rejection'
+  end
+
+  def then_i_should_see_reasons_for_rejection_metrics
+    then_i_should_see_reasons_for_rejection_course_full
+    and_i_should_see_reasons_for_rejection_candidate_behaviour
+    and_i_should_see_reasons_for_rejection_honesty_and_professionalism
+    and_i_should_see_reasons_for_rejection_interested_in_future_applications
+    and_i_should_see_reasons_for_rejection_offered_on_another_course
+    and_i_should_see_reasons_for_rejection_other_advice_or_feeback
+    and_i_should_see_reasons_for_rejection_performance_at_interview
+    and_i_should_see_reasons_for_rejection_qualifications
+    and_i_should_see_reasons_for_rejection_quality_of_application
+    and_i_should_see_reasons_for_rejection_safeguarding_concerns
+  end
+
+private
+
+  def reject_application_with_structured_reasons(application_choice)
+    application_choice.update!(
+      status: :rejected,
+      structured_rejection_reasons: {
+        course_full_y_n: 'Yes',
+        candidate_behaviour_y_n: 'Yes',
+        honesty_and_professionalism_y_n: 'Yes',
+        performance_at_interview_y_n: 'Yes',
+        qualifications_y_n: 'Yes',
+        quality_of_application_y_n: 'Yes',
+        safeguarding_y_n: 'Yes',
+        offered_on_another_course_y_n: 'Yes',
+        interested_in_future_applications_y_n: 'Yes',
+        other_advice_or_feedback_y_n: 'Yes',
+      },
+      rejected_at: Time.zone.now,
+    )
+  end
+
+  def reject_application_without_structured_reasons(application_choice)
+    application_choice.update!(
+      status: :rejected,
+      rejected_at: Time.zone.now,
+    )
+  end
+
+  def then_i_should_see_reasons_for_rejection_course_full
+    expect(all('.govuk-heading-m')[0]).to have_content('Course full')
+    expect(all('.govuk-grid-row')[0]).to have_content('10% of all rejections')
+    expect(all('.govuk-grid-row')[1]).to have_content('2 total')
+    expect(all('.govuk-grid-row')[1]).to have_content('1 this month')
+  end
+
+  def and_i_should_see_reasons_for_rejection_candidate_behaviour
+    expect(all('.govuk-heading-m')[1]).to have_content('Candidate behaviour')
```

Comment:
The index numbers here might make the tests a bit brittle, alternative would be to put a unique `id` attribute on each section and use something like `within('#id').do ... end`

Reply:
@stevehook We initially were using `id` as an identifier in our tests, but after a discussion with @davidgisbey we decided that these tests should be more granular. Granted Paul's recent changes to the front end broke the tests (which are now fixed)because of this, but arguably this shouldn't happen again. @davidgisbey is it worth reverting to `id` here?

Reply:
I don't think this should hold up this PR.